server/monitor.go

```diff
@@ -0,0 +1,504 @@
+// Copyright 2017 Apcera Inc. All rights reserved.
+
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"runtime"
+	"sort"
+	"strconv"
+	"time"
+
+	gnatsd "github.com/nats-io/gnatsd/server"
+	"github.com/nats-io/nats-streaming-server/stores"
+)
+
+// Routes for the monitoring pages
+const (
+	RootPath     = "/nss"
+	ServerPath   = RootPath + "/server"
```

Comment:
should these names follow the same convention(z) as gnatsd (`varz`, `storez`, `clientz`, `channelz`)?